service/worker/scanner/replication/replication.go

```diff
@@ -32,7 +32,6 @@ import (
 
 	apicommon "go.temporal.io/api/common/v1"
 	"go.temporal.io/api/enums/v1"
-	apinamespace "go.temporal.io/api/namespace/v1"
```

Comment:
I see it is not part of this PR but I somehow overlooked it before. Can you use same import aliases as we use everywhere in the project:
`apicommon` -> `commonpb`
`apireplication` -> `replicationpb`
`"go.temporal.io/api/enums/v1"` -> `enumspb "go.temporal.io/api/enums/v1"`

Reply:
This probably needs to be fixed everywhere else.